superset/viz.py

```diff
@@ -2124,8 +2131,24 @@ def process_spatial_data_obj(self, key, df):
         return df
 
     def query_obj(self):
-        d = super(BaseDeckGLViz, self).query_obj()
         fd = self.form_data
+
+        # add NULL filters
+        if fd.get('filter_nulls'):
+            spatial_columns = set()
```

Comment:
Can we get a unit test around this block? Just making sure `fd` has the proper values

Reply:
Will do! +1